src/gui/qjacktrip.cpp

```diff
@@ -240,8 +240,9 @@ QJackTrip::QJackTrip(Settings* settings, bool suppressCommandlineWarning, QWidge
     connect(m_netManager.data(), &QNetworkAccessManager::finished, this,
             &QJackTrip::receivedIP);
     // Use the ipify API to find our external IP address.
-    m_netManager->get(QNetworkRequest(QUrl(QStringLiteral("https://api.ipify.org"))));
-    m_netManager->get(QNetworkRequest(QUrl(QStringLiteral("https://api6.ipify.org"))));
+    // m_netManager->get(QNetworkRequest(QUrl(QStringLiteral("https://api.ipify.org"))));
+    m_netManager->get(
```

Comment:
Does this change anything about how IPv4 vs IPv6 is handled?

Reply:
It sure does. Not a fan. My preferred solution would be to have the existing code run on first display of the standard interface instead so that we retain the existing functionality without affecting vs users. I can make the required changes tonight when I get home from the tour I'm currently on.

Reply:
+1 to just moving this into classic interface (only)